lib/modules/accountSwitcher.js

```diff
@@ -305,13 +330,15 @@ export function closeAccountMenu() {
 
 function findMatchingAccount(username, partialMatch) {
 	const accounts = module.options.accounts.value;
-	const matched = accounts.find(([user]) => user.toUpperCase() === username.toUpperCase());
+	const constUsername = username;	// used to please flow tests
```

Comment:
This change not necessary if checking that `thing.getAuthor()` is not null (like how `loggedInUser()` is checked).

Reply:
Interestingly, that's what I tried first but after some googling it seems that flow does not like checking directly against a function argument but requires that the variable is a `const`. So I cannot just do `if (!username) return;` like I do below.

Reply:
That's a little weird. See if the Flow error changes by typing the username parameter (`function findMatchingAccount(username: string, partialMatch: boolean) {`.